homework/05LessonFive.js

```diff
@@ -63,15 +63,15 @@ for (forLoopCounter = 0; forLoopCounter > -10; forLoopCounter--) {
 
 // Problem 8:
 // Fill in forLoopArrayResult so that it matches forLoopArray
-var forLoopArrayResult = [];
+var forLoopArrayResult = [0, -1, -2, -3, -4, -5, -6, -7, -8, -9];
 
 // ****
 // Concept Checkpoint
 //
 // Write your answer in comments
 //
 // What are loops? Why do we use them?
-//
+// Loop provides a way to go through a function/method using certain collection of values in an incremental/decremental order 
```

Comment:
Pretty much. 

It's a way to repeat a method or run logic until a condition is met.